portal/portal-service-impl/impl/src/java/org/sakaiproject/portal/service/SiteNeighbourhoodServiceImpl.java

```diff
@@ -590,10 +583,7 @@ public String parseSiteAlias(String alias)
 		}
 		catch (IdUnusedException e)
 		{
-			if (log.isDebugEnabled())
-			{
-				log.debug("No alias found for "+ id);
-			}
+            log.debug("No alias found for {}", id);
```

Comment:
The indentation here looks wrong, could you try to match it with the surrounding context, the issue maybe because of differences tabs and spaces.